OnlinerTests/Tests/BaseTest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using NUnit.Framework;
+using OnlinerTests.PageObjects.Basic;
+using OpenQA.Selenium;
+
+namespace OnlinerTests.Tests
+{
+    class BaseTest
+    {
+        protected object[] data;
+
+        [SetUp]
+        public void AhSHereWeGoAgain()
```

Comment:
1. give the method an appropriate name
2. it would be better to start a driver before each suite rather than test

Reply:
I changed the name, but I'm not sure why driver should start before each suite? Wouldn't I need to "reset" it between each test?